tests/testcases/test_filter_end/expected.vcf

```diff
@@ -0,0 +1,44 @@
+##fileformat=VCFv4.1
+##FILTER=<ID=PASS,Description="All filters passed">
+##fileDate=20210218
+##contig=<ID=chr1,length=1>
```

Comment:
chromosome length should be compatible with the positions specified in the records

Reply:
all fixed